membership_variable_period/models/product_product.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Antonio Espinosa <antonio.espinosa@tecnativa.com>
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openerp import models, api
+
+
+class ProductProduct(models.Model):
+    _inherit = "product.product"
+
+    @api.multi
+    def _get_next_date(self, date, qty=1):
```

Comment:
OK, it's the inheritance chain order problem that v10 solves. We have to stay as is now. Merging...